js/tests/unit/modal.js

```diff
@@ -167,7 +167,7 @@ $(function () {
         assert.notEqual($('#modal-test').length, 0, 'modal insterted into dom')
         $('.contents').click()
         assert.ok($('#modal-test').is(':visible'), 'modal visible')
-        $('#modal-test .modal-backdrop').click()
```

Comment:
Forgive my confusion, but why isn't this `$('.modal-backdrop').click()`? Isn't clicking on the backdrop rather than the modal itself the whole point of this test?